src/main/java/seedu/edrecord/model/person/Phone.java

```diff
@@ -12,7 +12,7 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Phone numbers should only contain numbers, and it should be exactly 8 digits long";
```

Comment:
Message should be updated to 8-16 digits too.

Reply:
Updated, thanks for catching it